tldw_Server_API/app/services/admin_settings_service.py

```
@@ -82,3 +85,108 @@ async def set_notes_title_settings(payload: NotesTitleSettingsUpdate) -> dict[st
+            cap = int(candidate.get("cap", defaults["cap"]))
+        except (TypeError, ValueError):
+            cap = defaults["cap"]
+
+        validated[key] = {
+            "weight": max(0, min(10, weight)),
+            "cap": max(0, min(100, cap)),
+        }
+    return validated
+
+
+async def _ensure_admin_settings_table():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_ensure_admin_settings_table`

(ANN202)